template-sample-app/template/src/config/store.js

```diff
@@ -71,20 +71,33 @@ const staticReducers = {
   app: appSlice,
 };
 
+function getCustomMiddleware() {
+  const devConfig = {
+    immutableCheck: {
+      ignoredPaths: ['carto.viewportFeatures'],
+    },
+    serializableCheck: {
+      ignoredPaths: ['carto.viewportFeatures'],
+      ignoredActions: ['carto/setViewportFeatures'],
+    },
+  };
+
+  const prodConfig = {
+    immutableCheck: false,
+    serializableCheck: false,
+  };
+
+  const isProductionEnv = process.env.NODE_ENV !== 'production';
```

Comment:
Should be `process.env.NODE_ENV === 'production'` or am I missing something?

Reply:
🤦🏻 .... No, the only missing thing is that I was tired... thx!

Reply:
Approve it if you want, please